code/game/objects/items/food/misc.dm

```diff
@@ -34,7 +34,7 @@
 	tastes = list("cheese" = 1)
 	foodtypes = DAIRY
 	w_class = WEIGHT_CLASS_SMALL
-	var/rat_heal = 10
+	var/rat_heal = 10	// used to determine how much health rats/regal rats recover when they eat it.
```

Comment:
This isn't a doc comment, this is just a comment.

Doc comments precede the variable, and use `///`.